mesheryctl/internal/cli/root/system/restart.go

```diff
@@ -97,3 +97,7 @@ func restart() error {
 	}
 	return nil
 }
+
```

Comment:
It works correctly on the Docker platform, there's not a need for `skip-update` for Kubernetes in `system restart`. 
Nice work, @alphaX86